docs/docsite/rst/installation_guide/intro_installation.rst

```diff
@@ -145,7 +149,7 @@ Installing Ansible in a virtual environment with ``pip``
 
 .. note::
 
-	If you have Ansible 2.9 or older installed, you need to use ``pip uninstall ansible`` first to remove older versions of Ansible before re-installing it.
+	If you have Ansible 2.9 or older installed, you need to use ``pip uninstall ansible`` first to remove older versions of Ansible before re-installing it. Similarly, if you have installed Ansible 3, or ``ansible-base`` on its own, you need to use ``pip uninstall ansible-base`` before upgrading Ansible.
```

Comment:
We already have a section on Upgrading Ansible with `pip` - can we shorten these notes to point to that heading, then put whatever additional details we need in that section?

Reply:
good idea. added in latest commit.